src/tiled/objectselectiontool.cpp

```diff
@@ -825,6 +825,44 @@ void ObjectSelectionTool::updateHandleVisibility()
     for (int i = 0; i < AnchorCount; ++i)
         mResizeHandles[i]->setVisible(showHandles && mMode == Resize);
 
+    if (hasSelection) {
+        int selectedObjectsCount = mapDocument()->selectedObjects().count();
+        MapObject *firstSelectedObj =mapDocument()->selectedObjects().at(0);
+        qreal totalHeight = 0.0, totalWidth = 0.0;
```

Comment:
Summing up the totals won't work correctly. It's not the sum of the widths that matters, but the width of the bounding box around all the selected objects. You can see this being calculated in `ObjectSelectionTool::updateHandles` as `boundingRect`.